docs/user/addons.rst

```diff
@@ -0,0 +1,53 @@
+Read the Docs Addons
+====================
+
+**Read the Docs Addons** is a group of features for documentation readers and maintainers that you can add to any documentation set hosted on Read the Docs:
+
+:doc:`Traffic analytics </analytics>`
+    explore pageviews for your docs
+
+:doc:`DocDiff </pull-requests>`
+    highlight changed output from pull requests
+
+:doc:`Pull request notification </pull-requests>`
+    notify readers that they are reading docs from a pull request
+
+:doc:`Flyout </flyout-menu>`
+    easily switch between versions and translations
+
+:doc:`Non-stable notification </notifications>`
+    notify readers that they are reading docs from a non stable release
+
+:doc:`Latest version notification </notifications>`
+    notify readers that they are reading docs from an older version
+
+:doc:`Search as you type </server-side-search/index>`
+    get search results faster
+
+Enabling Read the Docs Addons
+-----------------------------
+
+All projects using the ``build.command`` :ref:`build command <config-file/v2:build.commands>` are already using the Addons, other projects can enable them by following these steps:
+
```

Comment:
```suggestion
All projects using ``mkdocs`` :ref:`mkdocs <config-file/v2:mkdocs>` or the ``build.command`` :ref:`build command <config-file/v2:build.commands>` are already using the Addons, other projects can enable them by following these steps:
```

I forgot to mention that MkDocs is already using addons by default. We deployed this some weeks ago.

Reply:
I've been linking to here, instead of reproducing in various places :-)